HW_5/task_1.py

```diff
@@ -0,0 +1,22 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+# *Пример:*
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8 
+
+
+def stepen(num_a, num_b):
+    # num_a = int(input("Ведите число A: "))
+    # num_b = int(input("Ведите число B: "))
+    if num_b == 1:
+        return num_a
+    elif num_b == 0:
+        return " 1 "
```

Comment:
А почему бы просто число не вернуть? Почему нужно возвращать строку?
```suggestion
        return 1
```